Felienne/hedy
# [QoL] Removes cursor from readOnly ace editors

**Description**
In this PR we make a minor QoL fix by removing the cursor from all readOnly ace editors. As these are only used as code snippets and already contain a copy button the cursor is redundant and ugly as it is always shown before the first character.

**Fixes**
This PR fixes #2847.

**How to test**
Verify that in all readOnly ace editors the cursor is removed. See the difference below:

**With cursor**
![Schermafbeelding 2022-06-02 om 15 11 27](https://user-images.githubusercontent.com/38155424/171636988-f3912a4b-d3bb-4bb0-b110-e6de08bac13e.png)

**Without cursor**
![Schermafbeelding 2022-06-02 om 15 12 09](https://user-images.githubusercontent.com/38155424/171636996-f0010fc5-4af9-49d8-823c-5dd327fcdb6a.png)
